totalRP3/tools/Locale.lua

```diff
@@ -1266,6 +1266,7 @@ Please keep in mind that changing those settings might alter your experience wit
 	--- THEN MOVE IT UP ONCE IMPORTED
 	------------------------------------------------------------------------------------------------
 
+	CO_TOOLTIP_CURRENT_LINES = "Max \"current\" lines count",
```

Comment:
The meaning is kind of obscure. Does it means it limits the number of lines in a long text that wraps over multiple lines?
It should probably be "Max \"current\" line breaks".